docs/user/dashboard/dashboard.asciidoc

```diff
@@ -98,444 +98,141 @@ To add a saved panel, click *Add from library*, then select the panel you want t
 TIP: To access your saved panels, open the main menu, then click *Visualize*.
 
 [float]
-[[create-panels-with-lens]]
-== Tutorial: Create a dashboard of panels with web server data
```

Comment:
This was copy+pasted into a separate page. Seems a little easier to organize as an `include` if we want to move it back.